src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/listeners/CleanupVerifierListener.groovy

```diff
@@ -94,7 +95,7 @@ class CleanupVerifierListener extends AbstractSpringListener {
         regionVerifications.verify()
         withPool {
             database.getIsls(topology.isls).eachParallel {
-                assert it.timeUnstable == null
+                //assert it.timeUnstable == null
```

Comment:
This check is disabled, maybe we can get rid of it

Reply:
Thank you for noticing, we need this code. Uncommented it.